components/director/internal/domain/apptemplate/fixtures_test.go

```diff
@@ -169,6 +193,62 @@ func fixModelAppTemplateInput(name string, appInputString string) *model.Applica
 	}
 }
 
+func fixRegionalModelAppTemplateInputWithProductLabel(name, appInputString, region string) *model.ApplicationTemplateInput {
+	desc := testDescription
+
+	return &model.ApplicationTemplateInput{
+		Name:                 name,
+		Description:          &desc,
+		ApplicationNamespace: str.Ptr("ns"),
+		ApplicationInputJSON: appInputString,
+		Placeholders:         fixModelPlaceholdersWithRegion(),
+		Labels: map[string]interface{}{
+			"test":                  "test",
+			AppTemplateProductLabel: []interface{}{"role"},
+			RegionKey:               region,
+		},
+		AccessLevel: model.GlobalApplicationTemplateAccessLevel,
+	}
+}
+
+func fixGlobalModelAppTemplateInputWithProductLabel(name, appInputString string) *model.ApplicationTemplateInput {
```

Comment:
The `fixGlobalModelAppTemplateInputWithProductLabel` and `fixRegionalModelAppTemplateInputWithProductLabel` return the same input model with one difference. They can use one function returning the input model with the common parts and adjust only the labels value.